server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -96,6 +97,72 @@ public void teardown() {
     }
 
     @Test
+    public void setMailboxesShouldNotCreatedWhenOverLimitName() {
+        String overLimitName = StringUtils.leftPad("a", MailboxConstants.DEFAULT_LIMIT_MAILBOX_NAME_SIZE, "b");
```

Comment:
maybe StringUtils.repeat("a", MailboxConstants.DEFAULT_LIMIT_MAILBOX_NAME_SIZE) is more straightforward?